services/shell/server_test.go

```diff
@@ -0,0 +1,49 @@
+package shell_test
```

Comment:
The Shell service had no pre-existing `server_test`. I added one, but it only tests `DoCommand` functionality.